src/drivers/frsky_telemetry/sPort_data.c

```diff
@@ -223,3 +223,65 @@ void sPort_send_FUEL(int uart)
 	uint32_t fuel = (int)(100 * vehicle_status.battery_remaining);
 	sPort_send_data(uart, SMARTPORT_ID_FUEL, fuel);
 }
+
+void sPort_send_GPS_LON(int uart) {
+	/* send longitude */
+	/* get a local copy of the global position data */
+	struct vehicle_global_position_s global_pos;
+	memset(&global_pos, 0, sizeof(global_pos));
+	orb_copy(ORB_ID(vehicle_global_position), global_position_sub, &global_pos);
```

Comment:
Would it make sense to spend the static allocation to hold the struct and only update on sending lon?

Also in C++ this is easier than memset:
```
struct vehicle_global_position_s global_pos = {};
```